tensorboard/data/experimental/experiment_from_dev_test.py

```diff
@@ -0,0 +1,250 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tests for tensorboard.uploader.exporter."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import sys
+from unittest import mock
+
+import numpy as np
+import grpc
+import grpc_testing
+import tensorflow as tf
+
+from tensorboard.data.experimental import experiment_from_dev
+from tensorboard.uploader import test_util
+from tensorboard.uploader.proto import export_service_pb2
+from tensorboard.util import grpc_util
+
+
+class ExperimentFromDevTest(tf.test.TestCase):
```

Comment:
At a glance it doesn’t look like this uses any TensorFlow-specific
features; can it be a `tb_test.TestCase` instead, which runs
significantly faster due to skipping the TensorFlow import?